lib/datadog/appsec/contrib/rack/request_middleware.rb

```diff
@@ -13,7 +13,6 @@ module Datadog
   module AppSec
     module Contrib
       module Rack
-        # Create an array of lowercased headers
```

Comment:
Did you remove this comment because it was incorrect?

Reply:
No it is not incorrect but it doesn't add anything (it was originally saying more than that but I factorised the corresponding code, and the comment with it, which led to this not very useful comment that I forgot to delete) (https://github.com/DataDog/dd-trace-rb/pull/3528#discussion_r1537860909)